lua/lspconfig/server_configurations/denols.lua

```diff
@@ -5,7 +5,12 @@ local function buf_cache(bufnr, client)
   local params = {}
   params['referrer'] = { uri = vim.uri_from_bufnr(bufnr) }
   params['uris'] = {}
-  client.request_sync('deno/cache', params)
+  client.request('deno/cache', params, function(err, _result, ctx)
+    if err then
+      local uri = ctx.params.referrer.uri
```

Comment:
`uri` not make sense  convert to file name  by using `vim.uri_to_fname`

Reply:
ah oops

Reply:
changed